sqlglot/executor/env.py

```diff
@@ -151,6 +151,7 @@ def interval(this, unit):
     "CAST": cast,
     "COALESCE": lambda *args: next((a for a in args if a is not None), None),
     "CONCAT": null_if_any(lambda *args: "".join(args)),
+    "SAFECONCAT": null_if_any(lambda *args: "".join(args)),
```

Comment:
this needs to convert all args to strings

Reply:
ah missed it, thanks